js/ui/viewSelector.js

```diff
@@ -199,9 +314,99 @@ const ViewsDisplay = new Lang.Class({
     Name: 'ViewsDisplay',
 
     _init: function() {
+        this._enterSearchTimeoutId = 0;
+        this._localSearchMetricTimeoutId = 0;
```

Comment:
Again, same problem mentioned before with some weird squashing. I'll fix that up